Branding/Ghostery/de.lproj/PrivateBrowsing.strings

```diff
@@ -0,0 +1,17 @@
+/* Context menu option for opening a link in a new private tab */
+"ContextMenu.OpenInNewPrivateTabButtonTitle" = "In neuem Vergessen-Tab öffnen";
```

Comment:
```suggestion
"ContextMenu.OpenInNewPrivateTabButtonTitle" = "In neuem Ghost-Tab öffnen";
```

Reply:
Note for Cliqz: above we say "In neuem Vergessen-Modus Tab öffnen", here we say "In neuem Vergessen-Tab öffnen"—shouldn't it be the same?